declarations/WebpackOptions.d.ts

```diff
@@ -1206,6 +1206,10 @@ export interface OutputOptions {
 	 * Handles exceptions in module loading correctly at a performance cost.
 	 */
 	strictModuleExceptionHandling?: boolean;
+	/**
+	 * The name of the trusted types policy created by webpack to serve bundle chunks.
```

Comment:
consider mentioning that the default policy name is `"webpack"`

Reply:
Done :)